etc/helm/pachyderm/values.yaml

```diff
@@ -198,6 +198,7 @@ pachd:
   enterpriseLicenseKey: ""
   rootToken: ""
   enterpriseSecret: ""
+  oauthClientSecret: ""
```

Comment:
Need to run `make schema` to add this to the json schema

Reply:
It should be included in this PR